.github/workflows/docker.yml

```diff
@@ -1,12 +1,15 @@
-name: Docker
+name: Build
 
 on:
+  release:
+    types: [ published ]
   push:
     branches:
       - 'master'
   pull_request:
     types: [synchronize, opened, reopened, ready_for_review]
```

Comment:
Just a comment: `pull_request` trigger will use the PR SHA and run any changes made on any PR. PRs from forks don't contain secrets